tests/cypress.json

```diff
@@ -0,0 +1,5 @@
+{
+    "baseUrl": "http://php80/orangehrm/symfony/web/index.php",
```

Comment:
Shall we add this config change to readme

Reply:
done